### ISSUE TITLE
'INSTACART_STORE_LIST' missing

### ISSUE THREAD

**Issue by gperez1138** [rkadam/notify-me#4]
I was just trying to run a simple command `python instacart-delivery-time-alert.py --help` and 
got this issue:
  File "instacart-delivery-time-alert.py", line 36, in main
    store_list = os.getenv('INSTACART_STORE_LIST').split(',')
AttributeError: 'NoneType' object has no attribute 'split'

**Comment by rkadam** [comment]
My bad, I'd forgot to add this new variable into sample file.
I just added it 

INSTACART_STORE_LIST=costco,cvs

So now you can choose costco or cvs. Let me know how it goes.

**Comment by gperez1138** [comment]
quick question, I am new to APIs, what exactly do you put in INSTACART_COOKIE_CONTENT ? I was able to get postman to work and all but not sure what to put in for this variable

**Comment by rkadam** [comment]
Good question. Unless one authenticate themselves, application can't serve data related to that user. Since we don't have official documentation to log in as a user with API, we're doing (ethical) hack here. We're using Postman to get the session cookie. This cookie contains session id and all other information that Instacart application needs in order authenticate user as well as to get other information as well as such userid and so on. If you read cookie content you will see lot of key value pairs being included into it.

In a nutshell, it has all information that helps our script can login as you and make API calls to retrieve information on your behalf just as the way Browser receives information when you interact with Instacart Web UI or Mobile App

Hope this helps.
Raju

**Comment by gperez1138** [comment]
Nice got it to work.
I had to add this though. I don't know why on my machine it won't locate the .env file
```
from pathlib import Path
env_path = Path('.') / 'example.env'
load_dotenv(dotenv_path=env_path)
`` `

**Comment by rkadam** [comment]
Good to hear things are working for you now!

As per [dotenv documentation](https://pypi.org/project/python-dotenv/) you can put .env file in current directory or any other directory mentioned in PATH variable

> The easiest and most common usage consists on calling load_dotenv when the application starts, which will load environment variables from a file named .env in the current directory or any of its parents or from the path specified;

I would suggest you keep it in your working directory, and then let original code work as is
> load_dotenv(override=True)

That should do the trick to load .env file without mentioning of explicit path as you have described in code snippet.

**Comment by rkadam** [comment]
README and example.env files updated to address this issue. Calling it done!
